poll.js

```diff
@@ -0,0 +1,150 @@
+'use strict'
+var imgs = ['bag.jpg',
+            'banana.jpg',
+            'bathroom.jpg',
+            'boots.jpg',
+            'breakfast.jpg',
+            'bubblegum.jpg',
+            'chair.jpg',
+            'cthulhu.jpg',
+            'dog-duck.jpg',
+            'dragon.jpg',
+            'pen.jpg',
+            'pet-sweep.jpg',
+            'scissors.jpg',
+            'shark.jpg',
+            'sweep.png',
+            'tauntaun.jpg',
+            'unicorn.jpg',
+            'usb.gif',
+            'water-can.jpg',
+            'wine-glass.jpg'];
+
+var choices = [];
+choices.push(document.getElementById('picture0'));
+choices.push(document.getElementById('picture1'));
+choices.push(document.getElementById('picture2'));
+var results = document.getElementById('results');
+var rlist = document.getElementById('resultslist');
+
+/**
+ * [Image description]
+ * @param {[type]} file [description]
+ */
+function Image(file) {
+  this.score = 0; //times selected
+  this.appeared = 0; //times appeared on the screen
+  this.file = file; //image path
+}
+
+Image.prototype.getPath = function() {
+  return 'img/' + this.file;
+}
+
+Image.prototype.tally = function(chosen) {
+  this.appeared++;
+  if(chosen){this.score++;}
+}
+
+Image.prototype.getPercent = function() {
+  if(this.score === 0) return 0;
+  var prc = (this.score / this.appeared) * 100;
```

Comment:
Instead of saving yourself the time of typing a couple characters, why not take the time to give this a variable name that isn't a meaningless bundle of letters?